src/repository/accountRepository.js

```diff
@@ -61,7 +61,9 @@ module.exports.getUserProfile = async (userId) => {
         kycStatus: user.kycStatus,
         kycDocs: user.kycDocs,
         idmStatus: user.idmStatus,
-        purchases: purchases
+        purchases: purchases,
+        kycDetails: user.kycDetails,
```

Comment:
Sending user.kycDetails to the client might reveal information from IDM to the user, like why KYC failed. Is that wise?

Reply:
I see now it is a new field.